src/libraries/System.Text.Json/src/System/Text/Json/Serialization/JsonSerializerOptions.cs

```diff
@@ -20,7 +20,15 @@ public sealed partial class JsonSerializerOptions
     {
         internal const int BufferSizeDefault = 16 * 1024;
 
-        internal static readonly JsonSerializerOptions s_defaultOptions = new JsonSerializerOptions();
+        /// <summary>
+        /// Gets a read-only, singleton instance of <see cref="JsonSerializerOptions" /> that uses the default configuration.
+        /// </summary>
+        /// <remarks>
+        /// Each <see cref="JsonSerializerOptions" /> instance encapsulates its own converter cache,
```

Comment:
I don't think we need to be specific on "converter cache". Just "caches". It's not just converters, all of the JsonTypeInfo info is cached (properties, constructors, converters).

Reply:
Changed, although one could argue that JsonTypeInfo is just implementation detail for object converters. But I agree it should best be kept vague.